src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -134,17 +132,33 @@ private void extractPublisher() {
      * @param dcSchema Metadata in DublinCore format.
      */
     private void extractBibTexFields() {
-        List<String> relationships = dcSchema.getRelations();
-        if (relationships != null) {
-            for (String r : relationships) {
-                if (r.startsWith("bibtex/")) {
-                    r = r.substring("bibtex/".length());
-                    int i = r.indexOf('/');
-                    if (i != -1) {
-                        bibEntry.setField(r.substring(0, i), r.substring(i + 1));
+        Predicate<String> isBibTeXElement = s -> s.startsWith("bibtex/");
+        Consumer<String> splitBibTeXElement = s -> {
+            // split solution is complicated, because some fields contains url etc.
+            String temp = s.substring("bibtex/".length());
+            int i = temp.indexOf('/');
```

Comment:
Please add an example why you need this substring. It is clear from the code that it is bibtex/key/value, but it should be documented somewhere (and referenced from here).

Reply:
You are right with the schema bibtex/key/value. 
In a former approach I used String#split, but in some cases the value is an url or doi etc. 

So the value in this case can contain "/" and therefore the substring is the easiest way to handle this. Documenting the url problem would be beneficial for future changes.